rest-api-json-bulk/src/main/kotlin/AppMain.kt

```diff
@@ -0,0 +1,70 @@
+package org.modelix.sample.restapijsonbulk
+
+import MPSRemoteClient
+import University.Schedule.api.gen.University_Schedule
+import University.Schedule.api.gen.jetbrains_mps_lang_core
+import io.ktor.http.*
+import io.ktor.serialization.gson.*
+import io.ktor.server.application.*
+import io.ktor.server.engine.*
+import io.ktor.server.locations.*
+import io.ktor.server.netty.*
+import io.ktor.server.plugins.autohead.*
+import io.ktor.server.plugins.contentnegotiation.*
+import io.ktor.server.plugins.defaultheaders.*
+import io.ktor.server.routing.*
+import loadModelSafe
+import org.modelix.model.api.INode
+import org.modelix.model.api.INodeReference
+import org.modelix.mps.apigen.runtime.MPSLanguageRegistry
+import org.modelix.sample.restapijsonbulk.models.apis.DefaultApi
+import org.slf4j.LoggerFactory
+
+val logger = LoggerFactory.getLogger("main")
+
+suspend fun main() {
+    registerLanguages()
+    val host = System.getenv().getOrDefault("MPS_HOST", "localhost")
+    val port = System.getenv().getOrDefault("MPS_PORT", "63320")
+    val models = System.getenv().getOrDefault("MODELS_TO_LOAD", "University.Schedule.sandbox").split(",")
```

Comment:
Let's use the official configuration system of ktor instead manual environment variable handling:

https://ktor.io/docs/configurations.html#read-configuration-in-code

Reply:
https://youtrack.jetbrains.com/issue/KTOR-523
https://youtrack.jetbrains.com/issue/KTOR-815

seems to be not possible for us :/

Reply:
Aren't we in an application context there already?

Reply:
implemented